projects/packages/scheduled-updates/src/class-scheduled-updates-logs.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Scheduled Updates Logs
+ *
+ * @package automattic/scheduled-updates
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Scheduled_Update_Logs class
+ *
+ * This class provides a static methods to log/retrieve logs for scheduled updates.
+ */
+class Scheduled_Updates_Logs {
+	/**
+	 * The name of the WordPress option where the logs are stored.
+	 */
+	const OPTION_NAME = 'jetpack_scheduled_update_logs';
+
+	/**
+	 * The maximum number of runs to keep per schedule_id.
+	 */
+	const MAX_RUNS_PER_SCHEDULE = 2;
+
+	/**
+	 * Action constant representing the different kind of log message.
+	 */
+	const PLUGIN_UPDATES_START    = 'PLUGIN_UPDATES_START';
+	const PLUGIN_UPDATE_SUCCESS   = 'PLUGIN_UPDATE_SUCCESS';
+	const PLUGIN_UPDATE_FAIL      = 'PLUGIN_UPDATE_FAIL';
+	const PLUGIN_UPDATE_ROLLBACK  = 'PLUGIN_UPDATE_ROLLBACK';
+	const PLUGIN_UPDATES_COMPLETE = 'PLUGIN_UPDATES_COMPLETE';
```

Comment:
Let's use the three statuses that have already been used in WPCOM.

1. `success`
2. `failure-and-rollback`
3. `failure-and-rollback-fail`

What do you think?